src/exampleTypes/ResultTable/Header.js

```diff
@@ -63,6 +71,7 @@ let Header = ({
 }) => {
   let popover = React.useState(false)
   let adding = React.useState(false)
+  let [addColumnTriggerField, setAddColumnTriggerField] = React.useState(null)
```

Comment:
I don't think you need to use react state here because this is repeated for each header, so you can just reference field directly

Reply:
I needed state since the user clicks on the column and I need to remember which one he clicked on. Not sure what you mean by just referencing `field` since in the Nested Picker `field` is the one the user selected and not the one we need to remember in order to know where to place the new column. What am I missing?

Reply:
@stellarhoof helped here. I got it.